cmu_graphics/cmu_graphics.py

```diff
@@ -1325,9 +1325,20 @@ def setActiveScreen(screen, fromRunApp=False):
         )
     if (screen in [None, '']) or (not isinstance(screen, str)):
         raise Exception(f'{repr(screen)} is not a valid screen')
-    redrawAllFnName = f'{screen}_redrawAll'
-    if redrawAllFnName not in app._app.userGlobals:
-        raise Exception(f'Screen {screen} requires {redrawAllFnName}()')
+    
+    redrawAllFnNames = ['redrawAll']
+    redrawAllInCorrectLanguage = 'redrawAll'
+    for language, translations in shape_logic.TRANSLATED_USER_FUNCTION_NAMES.items():
+        if language == 'keys':
+            continue
+        for redrawAllTranslation in translations.get('redrawAll', []):
+            if redrawAllTranslation not in redrawAllFnNames:
+                redrawAllFnNames.append(redrawAllTranslation)
+                if language == shape_logic.cmuGraphicsLanguage:
+                            redrawAllInCorrectLanguage = redrawAllTranslation
+
+    if not any(f'{screen}_{fnName}' in app._app.userGlobals for fnName in redrawAllFnNames):
```

Comment:
I see tests ensuring that screens work in Spanish when everything is set up right. What do you think about adding a short test ensuring that this exception is raised when it should be?